cg/apps/housekeeper/hk.py

```diff
@@ -99,7 +127,7 @@ def add_file(
 
     def files(
         self, *, bundle: str = None, tags: List[str] = None, version: int = None, path: str = None
-    ) -> Query:
+    ) -> Iterable[File]:
```

Comment:
Query is the correct datatype this returns! (check what method it was calling calls returns)
The return type is wrong in Housekeeper but its not worth fixing it there atm